services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceParentHelixAdmin.java

```diff
@@ -2301,7 +2301,9 @@ public void updateStore(String clusterName, String storeName, UpdateStoreQueryPa
       setStore.storeName = storeName;
       setStore.owner = owner.map(addToUpdatedConfigList(updatedConfigsList, OWNER)).orElseGet(currStore::getOwner);
 
-      if (!currStore.isHybrid() && (hybridRewindSeconds.isPresent() || hybridOffsetLagThreshold.isPresent())) {
+      boolean isUpdateForStoreMigration = storeMigration.isPresent() && storeMigration.get();
```

Comment:
nit: can we use `boolean isUpdateForStoreMigration = storeMigration.orElse(false);`

Reply:
Done